apps/ejabberd/src/ejabberd_c2s.hrl

```diff
@@ -1,6 +1,6 @@
 -include_lib("ejabberd/include/mod_privacy.hrl").
 -define(GEN_FSM, p1_fsm_old).
-
+-xep([{xep, 198}, {version, "1.3"}]).
```

Comment:
Why in the .hrl file while other c2s XEP attributes are defined in the .erl file?